docker-entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+function stop {
+	kill $(ps -C ts3server -o pid= | awk '{ print $1; }')
+	exit
+}
+
+trap stop INT
+trap stop TERM
+
+test -d /data/files || mkdir -p /data/files && chown teamspeak:teamspeak /data/files
```

Comment:
The `teamspeak` user doesn't have permissions to create `/data` by default when just running the container with a volume that doesn't have the right permissions to begin with.  This should be fine if we create `/data/` in the `Dockerfile` and set the appropriate permissions to begin with.